Cargo.toml

```diff
@@ -36,7 +36,7 @@ num = "0.4.2"
 # benchmarks
 criterion = "0.5.1"
 iai-callgrind = "0.10.2"
-rand = "0.8.5"
+rand = "0.9.0-alpha.1"
```

Comment:
Why the need for `alpha`?

Reply:
The `0.9.0` of the rand crate is not out yet and their devs seem to have released an alpha for people to use. Now was a good time to upgrade the dependency so I decided to do it and process a first batch a breaking changes that will be in `0.9.0`